_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -557,10 +557,10 @@ resource "aws_ebs_encryption_by_default" "ebs_encryption" {
 === Logging
 In the Logging section, the Benchmark recommendations target the following services:
 
-* link:https://aws.amazon.com/cloudtrail/[AWS CloudTrail]
-* link:https://aws.amazon.com/config/[AWS Config]
-* KMS key rotation
-* VPC Flow logs
+* link:https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-user-guide.html[AWS CloudTrail]
+* link:https://docs.aws.amazon.com/config/latest/developerguide/WhatIsConfig.html[AWS Config]
+* link:https://docs.aws.amazon.com/kms/latest/developerguide/rotate-keys.html[KMS key rotation]
+* link:https://docs.aws.amazon.com/vpc/latest/userguide/flow-logs.html[VPC Flow Logs]
```

Comment:
also a nice touch!

Reply:
Yeah good idea, that makes them consistent!